tests/tt_metal/tt_metal/common/device_fixture.hpp

```diff
@@ -100,3 +100,14 @@ class BlackholeSingleCardFixture : public DeviceSingleCardFixture {
         this->create_devices();
     }
 };
+
+class ArchFixture : public ::testing::Test {
+   protected:
+    tt::ARCH arch_;
+
+    void SetUp() override {
+        this->arch_ = tt::get_arch_from_string(tt::test_utils::get_umd_arch_name());
```

Comment:
See the comment in the test regarding the need for this; but if we end up keeping this -- `arch_` as a data member is already disambiguated with the underscore suffix, can drop `this->`?